unsplahAPI/unsplahAPI/Authentification/Authentication/View/AuthenticationViewController.swift

```diff
@@ -9,42 +9,161 @@ import UIKit
+                NSAttributedString(
+                    string: "Log In",
+                    attributes: [
+                        NSAttributedString.Key.font: DesignedSystemFonts.bodyBold,
+                        NSAttributedString.Key.foregroundColor: DesignedSystemColors.primaryText
+                    ]),
+                for: .normal
+            )
+        }
+    }
+    @IBOutlet private weak var underButtonsLabel: UILabel! {
+        didSet {
+            underButtonsLabel.numberOfLines = 3
+            underButtonsLabel.attributedText = NSAttributedString(
+                string: "By proceeding you agree to some random terms of service and privacy policy",
+                attributes: [
+                    NSAttributedString.Key.font: DesignedSystemFonts.smallText,
+                    NSAttributedString.Key.foregroundColor: DesignedSystemColors.primaryInsignificantText
+            ]
+            )
+        }
+    }
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-       userCancelable = viewModel?.user.$authenticationState.sink { [weak self] autheticationState in
+        userCancelable = viewModel?.user.$authenticationState.sink { [weak self] autheticationState in
             switch autheticationState {
             case .unauthenticated:
-                //TODO: show auth screen
+                // TODO: show auth screen
                 self?.authenticationProcess(result: .failure(AuthenticationErrors.unableToIdentifyUser))
             case .authenticated:
                 self?.authenticationProcess(result: .success(self?.viewModel?.user))
             case .authenticating:
-                //TODO: perform default process
+                // TODO: perform default process
                 print(autheticationState)
             }
         }
     }
-    
-    func authenticationProcess(result: Result <User?, Error>) {
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.unitOpacity = 0
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        let transitionOfScrollViewUp: Double = 2.0
+        UIView.animate(withDuration: transitionOfScrollViewUp) { [weak self] in
+            guard let self else { return }
+            self.backgroundImageTopConstraint.constant = -280
+            self.backgroundImageBottomConstraint.constant = 280
+            homeScreenView.layoutIfNeeded()
+        }
+        UIView.animate(withDuration: 1.0, delay: transitionOfScrollViewUp) { [weak self] in
+            guard let self else { return }
+            self.unitOpacity = 1.0
+        }
+    }
+
+    func authenticationProcess(result: Result <User?, AuthenticationErrors>) {
         switch result {
-        case .success(let user):
+        case .success(let _):
             coordinator?.didFinishAuthentification()
         case .failure(let error):
             self.constructView()
```

Comment:
I don't think the bellow stackview should depend on authentication, authentication is done when button is tapped.